senza/error_handling.py

```diff
@@ -116,3 +127,20 @@ def __call__(self, *args, **kwargs):
         except Exception as e:
             # Catch All
             self.die_unknown_error(e)
+
+
+# Setup Sentry
```

Comment:
This comment is exactly the same text of the function it is commenting about. That kind of comment is just annoying. :disappointed: